tt_metal/hw/ckernels/wormhole_b0/metal/llk_api/llk_sfpu/ckernel_sfpu_remainder.h

```diff
@@ -0,0 +1,68 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "ckernel.h"
+#include "ckernel_defs.h"
+#include "noc_nonblocking_api.h"
+#include "ckernel_sfpu_converter.h"
+#include "ckernel_sfpu_recip.h"
+
+using namespace sfpi;
+
+namespace ckernel {
+namespace sfpu {
+
+template <bool APPROXIMATION_MODE, int ITERATIONS = 8>
+inline void calculate_remainder(const uint value, const uint recip) {
```

Comment:
Is there a reason this is called remainder rather than mod? In C there is a difference with negative values between these 2 operations, and this one seems to do a mod, not remainder

Reply:
We named this as remainder to match the torch API `torch.remainder`. There is also another api like `torch.fmod`, which we will implement next to this, we dont have api called `torch.mod`, we will see if is there any difference between this remainder and mod from other sources.

Reply:
ok can it be documented? I see that pytorch does  `a - a.div(b, rounding_mode="floor") * b`, it will be good if this added somewhere for users.